### PR TITLE
Create a github action to automatically build and test

### PR DESCRIPTION
**Pull request by flipbug**

### CODE REVIEW

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
+    - name: Install dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --configuration Release --no-restore
+    - name: Test
+      run: dotnet test --no-restore --verbosity normal
```
Comment (phkiener) [inline .github/workflows/dotnet-core.yml:25]: `dotnet test --configuration Release --no-build`? Also, why verbosity normal? I think the default (minimal) is enough output

File: .github/workflows/dotnet-core.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: .NET Core
```
[the remark this answers is not in the document]
Comment (flipbug) [reply .github/workflows/dotnet-core.yml]: True, I just added the default file without much further consideration :)

File: .github/workflows/dotnet-core.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
```
[the remark this answers is not in the document]
Comment (flipbug) [reply .github/workflows/dotnet-core.yml]: removed

File: .github/workflows/dotnet-core.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
```
[the remark this answers is not in the document]
Comment (flipbug) [reply .github/workflows/dotnet-core.yml]: Yes it seems that wildcards are supported. I've changed it accordingly

File: .github/workflows/dotnet-core.yml (hunk 1 above)
Comment (flipbug) [reply .github/workflows/dotnet-core.yml]: I replaced the default with your suggestions, since it makes more sense :)

**Pull request merged**